OpenRA.Game/Scripting/ScriptContext.cs

```diff
@@ -86,8 +86,8 @@ public abstract class ScriptGlobal : ScriptObjectWrapper
 		public ScriptGlobal(ScriptContext context)
 			: base(context)
 		{
-			// The 'this.' resolves the actual (subclass) type
-			var type = this.GetType();
+			// GetType resolves the actual (subclass) type
+			var type = GetType();
```

Comment:
I can still remember the giant WTF moment when I found that adding `this.` fixed the issues that I was having, so i'm pretty sure, yeah.